src/components/fields/AddressField.js

```diff
@@ -68,21 +77,22 @@ const AddressField = (
     return setIsValid(newIsValid);
   }, []);
 
-  const handleChange = useCallback(
-    ({ nativeEvent: { text } }) => {
-      onChange(text);
+  const handleChangeText = useCallback(
+    text => {
       validateAddress(text);
       expandAbbreviatedClipboard();
+      setInputValue(text);
+      onChangeText(text);
     },
-    [expandAbbreviatedClipboard, onChange, validateAddress]
+    [setInputValue, expandAbbreviatedClipboard, onChangeText]
```

Comment:
i'm getting lint warnings, `validateAddress` is missing

Reply:
Fixed